ols/src/llms/llm_loader.py

```diff
@@ -302,12 +301,12 @@ def _watson_llm_instance(self) -> LLM:
             "url": (
                 self.llm_params.get("url")
                 if self.llm_params.get("url") is not None
```

Comment:
we should not gets the url from llm_params, we should only get it from the config object or from some hardcoded default.